### PR TITLE
Remove self-healing section

### PR DESCRIPTION
**Pull request by arajasek**
This is not (yet) a thing.

Also moving up and slightly rewording the `Verifiable storage` section because IMO that is far and away the highlight of storing data on Filecoin.

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/about-filecoin/why-filecoin.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://networks.filecoin.io
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “their”.
  ```
  ...Choice of tradeoffs Users get to choose their own tradeoffs between cost, re...
                                             ^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ... group of data centers offered by their provider but can choose to store the...
                                             ^^^^^^^^
  ```
- Consider using “many”.
  ```
  ...ding service. The network is made up of many different computers run by many...
                                             ^^^^^^^^^^^^^^
  ```
- Consider using “many”.
  ```
  ...e up of many different computers run by many different people and organizati...
                                             ^^^^^^^^^^^^^^
  ```
- As an alternative to the over-used intensifier ‘very’, consider replacing this phrase.
  ```
  ...ically only store tiny amounts of data, very expensively. Filecoin can provi...
                                             ^^^^^^^^^^^^^^^^
  ```
- Consider using “several”.
  ```
  ...ovider. Applications wishing to support several different providers aren’t l...
                                             ^^^^^^^^^^^^^^^^^
  ```
- If this is a compound adjective that modifies the following noun, use a hyphen.
  ```
  ...ssible, at the lowest possible price. # Open source code The code that runs ...
                                             ^^^^^^^^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “their”.
  ```
  ...Storage providers don’t have to develop their own software for managing thei...
                                             ^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...ments made to Filecoin’s code. # Active community Filecoin has an active com...
                                             ^^^^^^^^^
  ```

**Review by johnnymatthews — COMMENTED** [review]
Looks solid.

File: docs/about-filecoin/why-filecoin.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ description: Explore the features that make Filecoin a compelling system for sto
 
 This is an overview of features offered by Filecoin that make it a compelling system for storing files. This overview is intended for people already involved in large-scale data storage, for example, cloud storage or peer-to-peer storage networks.
 
+### Verifiable storage
+
+Filecoin generates verifiable traces that files have been stored correctly over time. Every storage provider proves that they are maintaining their files in every 24-hour window. Clients can efficiently scan these traces to confirm that their files have been stored correctly, even if the client was offline at the time. Any observer can check any storage provider’s track record and will notice if the provider has been faulty or offline in the past.
```
[the remark this answers is not in the document]
Comment (johnnymatthews) [reply docs/about-filecoin/why-filecoin.md:12]:
```suggestion
Filecoin has built-in processes to check the history of files and verify that they have been stored correctly over time. Every storage provider proves that they are maintaining their files in every 24-hour window. Clients can efficiently scan this history to confirm that their files have been stored correctly, even if the client was offline at the time. Any observer can check any storage provider’s track record and will notice if the provider has been faulty or offline in the past.
```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/about-filecoin/why-filecoin.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://networks.filecoin.io
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “their”.
  ```
  ...Choice of tradeoffs Users get to choose their own tradeoffs between cost, re...
                                             ^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ... group of data centers offered by their provider but can choose to store the...
                                             ^^^^^^^^
  ```
- Consider using “many”.
  ```
  ...ding service. The network is made up of many different computers run by many...
                                             ^^^^^^^^^^^^^^
  ```
- Consider using “many”.
  ```
  ...e up of many different computers run by many different people and organizati...
                                             ^^^^^^^^^^^^^^
  ```
- As an alternative to the over-used intensifier ‘very’, consider replacing this phrase.
  ```
  ...ically only store tiny amounts of data, very expensively. Filecoin can provi...
                                             ^^^^^^^^^^^^^^^^
  ```
- Consider using “several”.
  ```
  ...ovider. Applications wishing to support several different providers aren’t l...
                                             ^^^^^^^^^^^^^^^^^
  ```
- If this is a compound adjective that modifies the following noun, use a hyphen.
  ```
  ...ssible, at the lowest possible price. # Open source code The code that runs ...
                                             ^^^^^^^^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “their”.
  ```
  ...Storage providers don’t have to develop their own software for managing thei...
                                             ^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...ments made to Filecoin’s code. # Active community Filecoin has an active com...
                                             ^^^^^^^^^
  ```

**Pull request merged**